test/e2e/clusterclass_changes_test.go

```diff
@@ -43,6 +43,15 @@ var _ = Describe("When testing ClusterClass changes [ClusterClass]", func() {
 					"spec.template.spec.numCPUs": int64(4),
```

Comment:
Let's drop this here. It's enough if we set it below

Reply:
Forgot to drop it😂 Thanks!